kernel/src/process/program_loader/shebang.rs

```diff
@@ -23,14 +23,44 @@ pub fn parse_shebang_line(file_first_page: &[u8]) -> Result<Option<Vec<CString>>
     let shebang_header = &file_first_page[2..first_line_len];
     let mut shebang_argv = Vec::new();
     for arg in shebang_header.split(|&c| c == b' ') {
```

Comment:
Linux uses [`spacetab`](https://github.com/torvalds/linux/blob/8f5ae30d69d7543eee0d70083daf4de8fe15d585/fs/binfmt_script.c#L18) here. We may consider to use [`str::split_ascii_whitespace`](https://doc.rust-lang.org/std/primitive.str.html#method.split_ascii_whitespace). So we can avoid the ugly `arg.is_empty()` check?

Reply:
The issue is that the interpreter line is not required to be valid UTF‑8, so we cannot use `str` methods on it directly; otherwise we would need to convert with `str::from_utf8(shebang_header)`. In the recent Friday meeting we agreed that FsPath should accept `CStr/CString` (or similar) rather than `&str/String`.

I changed the split condition to split(|&c| c == b' ' || c == b'\t') so the behavior matches Linux (treating both spaces and tabs as separators).